src/middlewared/middlewared/plugins/account.py

```diff
@@ -496,16 +496,20 @@ def do_home_copy():
 
         return pk
 
-    @accepts()
-    def shell_choices(self):
+    @accepts(Int('oid', default=None))
```

Comment:
Minor nit, `oid` name feels odd, is it used anywhere else? As its shown in the docs I thought we could say simply `id` or `user_id` ?

Reply:
Updated :)